### ISSUE TITLE
${moduleName} always resolves to "{MyComponent}" instead of "MyComponent"

### ISSUE THREAD

**Issue by brianespinosa** [HardikModha/vscode-create-tests#3]
Right now I cannot make use of the templates for more complex use cases because the template tag for moduleName ALWAYS renders the opening and closing brackets with the string.

Here is what I was hoping to do:
```
[
  "import {felaSnapshot} from 'esp-style-provider';"
  "import ${moduleName} from './${modulePath}'"
  "",
  "describe('${moduleName}', () => {",
  "  test('DESCRIPTION', () => {",
  "    const snapshot = felaSnapshot(",
  "      <${moduleName} />"
  "    );",
  "    expect(snapshot.component).toMatchSnapshot();",
  "    expect(snapshot.styles).toMatchSnapshot();",
  "  })",
  "})",
  ""
]
```

I think this: `"import ${moduleName} from './${modulePath}'"`
should actually be: `"import {${moduleName}} from './${modulePath}'"`

If moduleName always returns the brackets in the string it is not usable in describe() and also in defining the component "<${moduleName} />". This `<${moduleName} />` becomes `<{MyComponent} />`

**Comment by hardikmodha** [comment]
@brianespinosa Yes, you are right. Thanks for reporting! Fix is on the way.

**Comment by hardikmodha** [comment]
@brianespinosa Published as [v1.0.1](https://github.com/HardikModha/vscode-create-tests/releases/tag/v1.0.1). Thanks for helping me improve vscode-create-tests! 😊

**Comment by brianespinosa** [comment]
Sweet. Just tested and it works perfectly. 👍 

Great work on the plugin. This is going to be a huge time saver for my team.
